data/dapps/airpuff.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't want to add a lot of fields to these and cause breaking changes later, only the alias is enough for now

Reply:
We should at least add further identifying information in case when the aliases are no longer enough. That's nowhere near the case yet.

Reply:
why would chainId be a breaking change ?

Reply:
I mean adding unneeded fields here and removing them later would be a breaking change so I won't add them until we're sure that we'll need them

Reply:
Tracked in https://github.com/api3dao/contracts/issues/250